mosaic/polypcolor.py

```diff
@@ -0,0 +1,75 @@
+from matplotlib.axes import Axes
+from matplotlib.collections import PolyCollection
+from matplotlib.colors import Normalize, Colormap
+from mosaic.descriptor import Descriptor
+
+from numpy.typing import ArrayLike
+
+from xarray.core.dataarray import DataArray
+
+
+def polypcolor(
+    ax: Axes,
+    descriptor: Descriptor,
+    c: ArrayLike,
+    alpha: float = 1.0,
+    norm: str | Normalize | None = None, 
+    cmap: str | Normalize | None = None,
+    vmin: float | None = None,
+    vmax: float | None = None,
+    facecolors: ArrayLike | None = None,
+    **kwargs
```

Comment:
Some explanation is needed in the docstring about what these `kwargs` get passed on to (`PolyCollection()`).